networkx/algorithms/dag.py

```diff
@@ -79,172 +82,184 @@ def is_directed_acyclic_graph(G):
     if not G.is_directed():
         return False
     try:
-        topological_sort(G, reverse=True)
+        consume(topological_sort(G))
         return True
     except nx.NetworkXUnfeasible:
         return False
 
 
-def topological_sort(G, nbunch=None, reverse=False):
```

Comment:
I don't see how to do it with this algorithm, but I think `topological_sort(G.subgraph(nx.descendants(G, nbunch)))` would work for single nodes?